apps/aecore/test/aec_chain_state_tests.erl

```diff
@@ -372,17 +386,17 @@ broken_chain_test_() ->
 
 broken_chain_postponed_validation() ->
     MainBC = gen_block_chain_by_target([?GENESIS_TARGET, 2, 5], 111),
-    AltChain = [B0, B1, B2, B3] = gen_block_chain_by_target([?GENESIS_TARGET, 2, 1], 222),
+    AltChain = [{_B0,S0}, {B1,S1}, {B2,S2}, {B3,S3}] = gen_block_chain_by_target([?GENESIS_TARGET, 2, 1], 222),
```

Comment:
The S* are unused, right? Better to have a util function that doesn't return these anyway.